rook/Tester.py

```diff
@@ -221,12 +224,28 @@ def _get_gold_files(self):
     """
       returns a list of the full path to the gold files
       @ In, None
-      @ Out, _get_gold_files, List(Strings), the path of the gold files.
+      @ Out, paths, List(Strings), the paths of the gold files.
     """
-    if len(self.specs['gold_files']) > 0:
+    this_OS = platform.system().lower()
+    available_OS = ['windows', 'mac', 'linux'] # list of OS with specific gold file options
```

Comment:
I'm assuming rook doesn't have to follow RAVEN's camelCase requirements?

Reply:
I think so. I was trying to match the coding style I saw in the existing code.